### DISCUSSION TITLE
Two Potentiometers without multiplexer

### DISCUSSION THREAD

**Discussion by jakit2015** [tttapa/Control-Surface#1077] [Q&A]
Hi, I'm a begginer in programming and I'm trying to build a MIDI controller to use with Amplitube and Arduino.
The Control-Surface was a welcome help and I managed to get the potentiomenter working.
But, when I tried using two potentiometers, it seems like one were conflicting with another.
By the example usage, the Example 2 use something called analog multiplexer wich I don't have.
Would you care to explain how to use two potentiometers?
What I've tried so far was defining both separately and as array (like the example 2)
´
CCPotentiometer pot1 =  { A4, MIDI_CC::General_Purpose_Controller_1 };
CCPotentiometer pot2 =  { A5, MIDI_CC::General_Purpose_Controller_2 };
´
But once the programming loads, the second one doesn't work. Is this the correct way of use for two potentiometers without the multiplexer?

**Comment by tttapa** [comment]
What you describe should work. See for example the [Multiple-Control-Change-Potentiometers](https://tttapa.github.io/Control-Surface-doc/Doxygen/d3/d8d/Multiple-Control-Change-Potentiometers_8ino-example.html) example.

You may want to try the "analog read serial" example that comes with the Arduino IDE to make sure that you're able to get meaningful values from the potentiometer.

**Reply by jakit2015** [reply]
thank you very much for the help
